soapec/2.03/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         soapec
+# Software Version: 2.03
+# Description:      a correction tool for SOAPdenovo
+# Website:          http://soap.genomics.org.cn/soapdenovo.html
+# Tags:             Genomics
+# Provides:         soapec 2.03
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/soapec 2.03/.
+# Pull Cmd:         docker pull biodckrdev/soapec
+# Run Cmd:          docker run --rm -it biodckrdev/soapec <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)